db/column_family.cc

```diff
@@ -1320,6 +1318,8 @@ void ColumnFamilyData::InstallSuperVersion(
       sv_context->superversions_to_free.push_back(old_superversion);
     }
   }
```

Comment:
The change here prevents `GetThreadLocalSuperVersion()` from returning staled SV compared to `GetSuperVersionNumber()`, but it is not necessarily needed to avoid the assertion error. Open to leave it unchanged.